Cargo.toml

```diff
@@ -1,28 +1,24 @@
 [workspace]
 resolver = "2"
 members = [
-    "gateway/crates/*",
-    "cli/crates/*",
-    "engine/crates/*",
-    "engine/crates/engine-v2/auth",
-    "engine/crates/engine-v2/config",
-    "engine/crates/engine-v2/axum",
-    "engine/crates/engine-v2/codegen",
-    "engine/crates/engine-v2/walker",
-    "engine/crates/engine-v2/schema",
-    "engine/crates/engine-v2/query-solver",
-    "engine/crates/engine-v2/id-derives",
-    "engine/crates/engine-v2/id-newtypes",
-    "engine/crates/engine-v2/parser",
-    "engine/crates/engine-v2/value",
-    "graph-ref",
-    "graphql-lint",
-    "gqlint",
-    "workspace-hack",
+    "cli",
+    "gateway",
+    "crates/*",
+    "crates/engine-v2/auth",
+    "crates/engine-v2/config",
+    "crates/engine-v2/axum",
+    "crates/engine-v2/codegen",
+    "crates/engine-v2/walker",
+    "crates/engine-v2/schema",
+    "crates/engine-v2/query-solver",
+    "crates/engine-v2/id-derives",
+    "crates/engine-v2/id-newtypes",
+    "crates/engine-v2/parser",
+    "crates/engine-v2/value",
 ]
 exclude = [
-    "engine/crates/gateway-v2",
-    "engine/crates/wasi-component-loader/examples",
+    "crates/gateway-v2",
```

Comment:
can we call this just gateway?

Reply:
that line is dead code, I just removed it.